sinks/api/supported_metrics.go

```diff
@@ -255,7 +256,50 @@ var statMetrics = []SupportedStatMetric{
 		},
 		OnlyExportIfChanged: true,
 	},
-
+	{
+		MetricDescriptor: MetricDescriptor{
+			Name:        "cpu/node_usage",
+			Description: "Cumulative CPU usage on all cores on a node",
+			Type:        MetricCumulative,
+			ValueType:   ValueInt64,
+			Units:       UnitsNanoseconds,
+		},
+		HasValue: func(spec *source_api.ContainerSpec) bool {
+			return spec.HasCpu && spec.HasResourceId
+		},
+		GetValue: func(spec *source_api.ContainerSpec, stat *cadvisor.ContainerStats) []internalPoint {
+			result := internalPoint{
+				value: int64(stat.Cpu.Usage.Total),
+				labels: map[string]string{
+					LabelComputeResourceID:   spec.ResourceId,
+					LabelComputeResourceType: "instance",
+				},
+			}
+			return []internalPoint{result}
+		},
+	},
+	{
+		MetricDescriptor: MetricDescriptor{
+			Name:        "memory/node_usage",
```

Comment:
`stat.Memory.Usage` is the entire's node's usage right? If your goal is to calculate system overhead, then it has to be `node_usage - usage of all containers under '/'`